demisto_sdk/commands/content_graph/objects/content_item.py

```diff
@@ -77,9 +84,16 @@ def in_pack(self) -> Optional["Pack"]:
         if in_pack := self.relationships_data[RelationshipType.IN_PACK]:
             return next(iter(in_pack)).content_item_to  # type: ignore[return-value]
         if pack_name := get_pack_name(self.path):
-            return BaseContent.from_path(
-                CONTENT_PATH / PACKS_FOLDER / pack_name
-            )  # type: ignore[return-value]
+            try:
+                return BaseContent.from_path(
+                    CONTENT_PATH / PACKS_FOLDER / pack_name
+                )  # type: ignore[return-value]
+            except InvalidContentItemException:
+                logger.warning(
```

Comment:
here it's a warning but in the other case it's an error - shouldn't it be the same in both?

Reply:
Notice this is `in_pack` propery inside a `content_item`. So it's not to bad to have a content item without a parent pack.